values.yaml

```diff
@@ -478,7 +478,12 @@ servicePreview:
 
 # Configure the ambassador agent
 agent:
-  enabled: false
+  enabled: true
+  # this will be empty when it first gets applied, then the user will edit the agent to
+  # make it start reporting
+  cloudConnectToken: ""
+  # TODO(acookin): replace with app.getambassador.io when released
+  rpcAddress: https://beta-app.datawire.io/
```

Comment:
I think we should use the real thing right away, and add override instructions where necessary.

Reply:
sounds good